packages/ui/src/components/avatar/avatar.native.tsx

```diff
@@ -1,11 +1,16 @@
-import { createBox } from '@shopify/restyle';
+import { type BoxProps } from '@shopify/restyle';
 
 import { type Theme } from '../../theme-native';
+import { Box } from '../box/box.native';
 
-export const Avatar = createBox<Theme>();
+interface AvatarProps extends BoxProps<Theme> {
```

Comment:
I could use `HasChildren` here but I don't want to create it a third time in `packages/ui`. 

Should we have a `types` package? Or where should we keep shared `types`?

Reply:
I just added it to `ui` in my approver work. Figure it's okay for UI package to have some helpers like `useOnMount` and type ones like HasChildren, [in the same vein Chakra has hook helpers](https://v2.chakra-ui.com/docs/hooks/use-boolean)

Reply:
OK, thanks!